### DISCUSSION TITLE
Mobile input methods, again

### DISCUSSION THREAD

**Discussion by heikomaru** [ppy/osu#14600] [Ideas] [answered]
I would like to know if you could add some extra buttons for the OSU modalities, it is difficult not to be able to make 6-star maps, or not to be able to do 6k in Osumania!

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/4201

**Comment by LittleEndu** [comment]
Please someone nuke the title of this discussion or at least edit it to be lowercase.

**Reply by bdach** [reply]
submitter is on cooldown anyways
